mlflow/tracing/constant.py

```diff
@@ -26,3 +27,6 @@ class SpanAttributeKey:
 
 # Trace request ID must have the prefix "tr-" appended to the OpenTelemetry trace ID
 TRACE_REQUEST_ID_PREFIX = "tr-"
+
+# A set of trace tags that cannot be updated by the user
+IMMUTABLE_TRACE_TAGS = {"mlflow.user", "mlflow.artifactLocation"}
```

Comment:
Can we import `MLFLOW_USER` from `mlflow/utils/mlflow_tags.py`?

Reply:
makes sense!